content/guides/building/compiling-arm.md

```diff
@@ -11,13 +11,13 @@ Haiku can be compiled for devices leveraging the ARMv7 or later processor archit
 
 ## Create a Compiler Toolchain
 
-Building the ARM compiler toolchain is quite easy using Haiku's ``configure`` tool.
+Building the ARM compiler toolchain is quite easy using Haiku's ```configure``` tool.
 
 > For a complete list of flags for the configure script, see [Haiku's Configure Options](/guides/building/configure)
 
 1. Perform a git clone haiku and buildtools
-2. Within the haiku source directory, create your workspace for ARM via ``mkdir generated.arm; cd generated.arm``
-2. Leverage configure to build your ARM toolchain. ``../configure -j2 --build-cross-tools arm ../../buildtools``
+2. Within the haiku source directory, create your workspace for ARM via ```mkdir generated.arm; cd generated.arm```
+2. Leverage configure to build your ARM toolchain. ```../configure -j2 --build-cross-tools arm ../../buildtools```
```

Comment:
You should be able to reduce these to just one set of backticks, not 3 on a side?

Reply:
3 is the standard for MD as far as I know:

https://help.github.com/articles/creating-and-highlighting-code-blocks/

Reply:
OK